core/modules/filters/strings.js

```diff
@@ -34,9 +34,35 @@ exports.titlecase = makeStringBinaryOperator(
 	function(a) {return [$tw.utils.toTitleCase(a)];}
 );
 
-exports.trim = makeStringBinaryOperator(
-	function(a) {return [$tw.utils.trim(a)];}
-);
+exports.trim = function(source,operator,options) {
+	var result = [],
+		suffix = operator.suffix || "",
+		operand = (operator.operand || ""),
+		fnCalc;
+	if (operand !== "") {
```

Comment:
Our coding style is to omit whitespace before `if` and the opening parenthesis.

Reply:
Assuming you mean between `if` and the parentheses, which I see from other .js files (now that I look) is the style of the rest of the code, I'll make that change.

Reply:
Done in commit 73c018eec.